src/index.ts

```diff
@@ -225,6 +178,28 @@ async function getCmaOutput(readLine, errorObj) {
   });
 }
 
+/**
+ * Invoke the task function and wrap the result as a Promise
+ *
+ * @param {Function} taskFunction - the task function to be invoked
+ * @param {Object} cumulusMessage - a full Cumulus message
+ * @param {Object} context - the Lambda context
+ * @returns {Promise} - the result of invoking the task function
+ */
+function invokePromisedTaskFunction(
```

Comment:
I'm not sure I understand what this is doing. Why not just call `taskFunction` directly?

Reply:
Because we cannot be certain taskFunction is async - this is just Yet Another Promisify Reimplementation.

That said: a8106eb

Reply:
Yeah, that comment was just terrible ^ this is resolved by another PR comment